opencti-platform/opencti-front/src/utils/filters/useSearchEntities.js

```diff
@@ -464,6 +464,28 @@ const useSearchEntities = ({
             unionSetEntities('objectContains', objectContainsEntities);
           });
         break;
+      case 'indicates':
+        fetchQuery(filtersStixCoreObjectsSearchQuery, {
+          types: (searchScope && searchScope.indicates) || [
+            'Stix-Core-Object',
+          ],
+          search: event.target.value !== 0 ? event.target.value : '',
+          count: 50,
+        })
+          .toPromise()
+          .then((data) => {
+            const indicatesEntities = R.pipe(
```

Comment:
Is there a way to not use Ramda ?

Reply:
You're right ! I'm going to remove ramda from this file !

Reply:
fixed